Analysis/config/2016/sources.cfg

```diff
@@ -21,19 +21,29 @@ regions: OS_Isolated OS_AntiIsolated SS_Isolated SS_AntiIsolated
 #limit_setup: MT2 MT2:2j1bR_noVBF,2j2b+R_noVBF,2j2Lb+B_noVBF,2j1b+_VBFL,2j1b+_VBFT
 #limit_setup: MVA mva_score:2j1bR_noVBF,2j2b+R_noVBF,2j2Lb+B_noVBF,2j1b+_VBFL,2j1b+_VBFT
 #limit_setup: kinFit_m kinFit_m:2j1bR_noVBF,2j2b+R_noVBF,2j2Lb+B_noVBF,2j1b+_VBFL,2j1b+_VBFT
-#limit_setup: DNN dnn_score+:2j1bR_noVBF,2j2b+R_noVBF,2j2Lb+B_noVBF,2j1b+_VBF
-limit_setup: m_bb m_bb:2j1bR_noVBF,2j2b+R_noVBF,2j2Lb+B_noVBF
-hist_cfg: hh-bbtautau/Analysis/config/histograms.cfg
+#limit_setup: DNN dnn_score+:2j1bR_noVBF,2j2b+R_noVBF,2j2Lb+B_noVBF,2j1b+_VBF_qqHH,2j1b+_VBF_ggHH,2j1b+_VBF_TT_L,2j1b+_VBF_TT_FH,2j1b+_VBF_TT,2j1b+_VBF_ttH,2j1b+_VBF_DY
+limit_setup: DNN dnn_score_kl1:2j1bR_noVBF,2j2b+R_noVBF,2j2Lb+B_noVBF,2j1b+_VBF_qqHH,2j1b+_VBF_ggHH,2j1b+_VBF_TT_L,2j1b+_VBF_TT_FH,2j1b+_VBF_TT,2j1b+_VBF_ttH,2j1b+_VBF_DY
```

Comment:
remove 2j1b+_VBF_TT_L and 2j1b+_VBF_TT_FH from the list (here and for the other years)